.gitignore

```diff
@@ -6,3 +6,6 @@ go-fdo-server
 go-fdo-server-*.tar.*
 rpmbuild
 test/workdir
+
+# Generated OpenAPI code
+generated/
```

Comment:
We need to track the changes in the generated code, we just cannot ignore the generated files